showcase/app/templates/page-components/advanced-table.hbs

```diff
@@ -45,15 +45,20 @@
     }}
   >
     <:body as |B|>
-      <B.Tr>
+      {{! @glint-expect-error }}
+      <B.Tr @selectionKey={{B.data.id}}>
```

Comment:
Opened a ticket for this, for some reason @selectionKey is required on the Tr. https://hashicorp.atlassian.net/browse/HDS-5167